### DISCUSSION TITLE
Append % to the name of Remainder node in Create Shader Node dialog

### DISCUSSION THREAD

**Discussion by Phoenixkaze** [godotengine/godot-proposals#9193] [Editor]
I was messing around with visual shader. When trying to add a common operator node like add, minus, multiply I can just type +, -, * to find them.

![image](https://github.com/godotengine/godot-proposals/assets/28686529/b7ab38a5-657c-46b1-915a-79c28290e23b)

Except the remainder/modulo operator. It does not have a % on its name. I was confused that I could not find it.

![image](https://github.com/godotengine/godot-proposals/assets/28686529/36571aa5-c638-445d-aaf6-367e030411bd)

Considering that GDScript and the shader language use % as the remainder operator and it lies under the operators section along with other operators, why not keep them consistent?

**Comment by Calinou** [comment]
I've opened a pull request for this: https://github.com/godotengine/godot/pull/88980
